### PR TITLE
Fix library name from expo-three to expo-three-ar

### PR DESCRIPTION
**Pull request by hamdigatri**
- Change library name to expo-three-ar.
- Fix links.

# Why

The API reference documentation for AR still reference to expo-three.

# How

Change library name to expo-three-ar and fix the links

# Test Plan

Read the API reference documentation for AR.

### CODE REVIEW

**Comment by cruzach** [comment]
Thanks! Could you make the same changes to the other versions of the docs? most importantly v36

**Comment by Gatreek92** [comment]
@cruzach done :)

File: docs/pages/versions/v36.0.0/sdk/AR.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ This API is pre-installed in [managed](../../introduction/managed-vs-bare/#manag
 import { AR } from 'expo';
 ```
 
-> [Examples can be found here](https://github.com/expo/expo-three/tree/master/example/screens/AR)
+> [Examples can be found here](https://github.com/expo/expo-three-ar/tree/master/example/screens/AR)
```
Comment (cruzach) [inline docs/pages/versions/v36.0.0/sdk/AR.md]: page not found

**Review by cruzach — CHANGES_REQUESTED** [review]
the example url in the pages (except `unversioned`) are 404s

File: docs/pages/versions/v36.0.0/sdk/AR.md (hunk 1 above)
Comment (Gatreek92) [reply docs/pages/versions/v36.0.0/sdk/AR.md]: Ah I see, i will fix it later today.

**Review by cruzach — APPROVED** [review]

**Pull request merged**